ignition/getting-started-with-ignition.md

```diff
@@ -0,0 +1,95 @@
+# Getting Started with Ignition #
+
+Before getting started, make sure that Ignition is supported on your platform.
+For a full list of supported platforms, refer [here][supported platforms].
+
+Ignition uses a JSON config to represent the set of changes to be made. The
+format of this config is detailed [here][config format]. One of the most
+important parts of this config is the version number. This **must** match the
+version number accepted by Ignition. If the config version isn't accepted by
+Ignition, Ignition will fail to run and prevent the machine from booting. This
+can be seen by inspecting the console output of the failed instance. For more
+information, check out the [troubleshooting](#Troubleshooting) section.
+
+Let's take a look at a few example configs.
+
+[supported platforms]: supported-platforms.md
+[config format]: https://github.com/coreos/ignition/blob/master/doc/ignition.md#configuration
+
+## Example Configs ##
+
+Each of these examples is written in version 1 of the config. Always double
+check to make sure that your config matches the version that Ignition is
+expecting.
+
+### BTRFS Root ###
+
+This config will find the device with the filesystem label "ROOT" (the root
+filesystem) and reformat it to BTRFS, maintaining the filesystem label. The
+force flag is needed here because CoreOS currently ships with an EXT4 root
+filesystem. Without this flag, Ignition would recognize that there is existing
+data and refuse to overwrite it.
+
+```json
+{
+	"ignitionVersion": 1,
+	"storage": {
+		"filesystems": [
+			{
+				"device": "/dev/disk/by-label/ROOT",
+				"format": "btrfs",
+				"create": {
+					"force": true,
+					"options": [
+						"--label=ROOT"
+						]
+				}
+			}
+		]
+	}
+}
+```
+
+### Metadata Agent Service ###
```

Comment:
Changing the title to reflect the topic that you're showing not the specific example.

Related to the above comment about the other title